Sources/BothamPullToRefreshHandler.swift

```diff
@@ -36,8 +39,14 @@ public class BothamPullToRefreshHandler: NSObject {
         self.refreshControl.endRefreshing()
     }
 
-    public func beginRefreshing(scrollView: UIScrollView) {
-        scrollView.setContentOffset(CGPoint(x: 0, y: -refreshControl.frame.size.height), animated: true)
-        self.refreshControl.beginRefreshing()
+    public func beginRefreshing() {
+        if !refreshControl.refreshing {
+            scrollView?.setContentOffset(CGPoint(x: 0, y: -refreshControl.frame.size.height), animated: true)
+            self.refreshControl.beginRefreshing()
+        }
+    }
+    
+    public func isRefreshing() -> Bool {
+        return self.refreshControl.refreshing
```

Comment:
Yes, we would need to bump the version, we have pending the release of Swift 2.2 version of BothamUI.